src/Snap/Internal/Http/Server.hs

```diff
@@ -155,11 +164,13 @@ httpServe :: Int                         -- ^ default timeout
           -> [ListenPort]                -- ^ ports to listen on
           -> ByteString                  -- ^ local hostname (server name)
           -> Maybe (ByteString -> IO ()) -- ^ access log action
+          -> Maybe AccessLogHandler
           -> Maybe (ByteString -> IO ()) -- ^ error log action
+          -> Maybe ErrorLogHandler
           -> ([Socket] -> IO ())         -- ^ initialisation
           -> ServerHandler               -- ^ handler procedure
           -> IO ()
-httpServe defaultTimeout ports localHostname alog' elog' initial handler =
+httpServe defaultTimeout ports localHostname alog' alh elog' elh initial handler =
```

Comment:
Oh, so you are saying that I can't backport this? I'm confused.

Looking at the exported method it is a completely different type (https://hackage.haskell.org/package/snap-server-0.9.5.1/docs/Snap-Http-Server.html#v:httpServe):

    httpServe :: Config Snap a -> Snap () -> IO ()

That is because httpServer is different to Int.httpServe in the public Server.hs file.

 * External: https://github.com/snapframework/snap-server/blob/0.9-stable/src/Snap/Http/Server.hs#L140
 * Internal: https://github.com/snapframework/snap-server/blob/0.9-stable/src/Snap/Http/Server.hs#L76

However, I have added records to the Config instance. Would that cause the same issue. I would hope not. :disappointed: